src/main/java/com/minecolonies/coremod/achievements/ModAchievements.java

```diff
@@ -39,279 +38,267 @@
-                                                                                                   ModBlocks.blockHutLumberjack,
-                                                                                                   achievementTownhall).registerStat();
+    public static final MineColoniesAchievement achievementBuildingLumberjack   = new MineColoniesAchievement("upgrade.lumberjack.first",
+            2,
+            11,
+            ModBlocks.blockHutLumberjack,
+            achievementTownhall);
     /**
      * Max out a lumberjack.
      */
-    public static final Achievement achievementUpgradeLumberjackMax = new MineColoniesAchievement("upgrade.lumberjack.max",
-                                                                                                   4,
-                                                                                                   11,
-                                                                                                   ModBlocks.blockHutLumberjack,
-                                                                                                   achievementBuildingLumberjack).registerStat();
+    public static final MineColoniesAchievement achievementUpgradeLumberjackMax = new MineColoniesAchievement("upgrade.lumberjack.max",
+            4,
+            11,
+            ModBlocks.blockHutLumberjack,
+            achievementBuildingLumberjack);
 
     /**
      * Upgrade a miner to lv 1.
      */
-    public static final Achievement achievementBuildingMiner   = new MineColoniesAchievement("upgrade.miner.first", 2, 8, ModBlocks.blockHutMiner,
-                                                                                              achievementTownhall).registerStat();
+    public static final MineColoniesAchievement achievementBuildingMiner   = new MineColoniesAchievement("upgrade.miner.first", 2, 8, ModBlocks.blockHutMiner, achievementTownhall);
     /**
      * Max out a miner.
      */
-    public static final Achievement achievementUpgradeMinerMax = new MineColoniesAchievement("upgrade.miner.max", 4, 8, ModBlocks.blockHutMiner,
-                                                                                              achievementBuildingMiner).registerStat();
+    public static final MineColoniesAchievement achievementUpgradeMinerMax = new MineColoniesAchievement("upgrade.miner.max", 4, 8, ModBlocks.blockHutMiner, achievementBuildingMiner);
 
     /**
      * Upgrade a fisher to lv 1.
      */
-    public static final Achievement achievementBuildingFisher   = new MineColoniesAchievement("upgrade.fisher.first", 2, 4, ModBlocks.blockHutFisherman,
-                                                                                               achievementTownhall).registerStat();
+    public static final MineColoniesAchievement achievementBuildingFisher   = new MineColoniesAchievement("upgrade.fisher.first", 2, 4, ModBlocks.blockHutFisherman, achievementTownhall);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 186 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00103)